Pipfile

```diff
@@ -8,6 +8,8 @@ verify_ssl = true
 [packages]
 django = "==2.2.6"
 django-environ = "==0.4.5"
+nodejs = "==0.1.1"
+npm = "==0.1.1"
```

Comment:
https://github.com/markfinger/python-nodejs
https://github.com/markfinger/python-npm

usuwaj to zanim zlozy jaja ;) nie wiem w ktorym momencie uczylem Cie zeby wykorzystywac paczki, ktore sa juz od dawna nieutrzymywane